rollup/default.ts.js

```diff
@@ -0,0 +1,86 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import path from 'path';
+import typescript from '@rollup/plugin-typescript';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+
+export default [
+	{
+		input: 'src/index.ts',
```

Comment:
Is there a way for us to reuse all of this that's repeated here so it's common and easier to maintain?

Reply:
Definitely but it might be too early for that.
I first want to see that these configs hold up for every plugin.
Only did some spot tests so I don't yet have a good view of all edge cases.

Having everything explicit makes it easier now to handle the unexpected.

I'll make an issue to clean this up later.

Reply:
Perfect!